docs/2.8.0/docs/canton/usermanual/console.rst

```diff
@@ -64,9 +64,37 @@ If you want to generate a skeleton remote configuration of a normal config file,
 
 However, you might have then to edit the config and adjust the hostname.
 
+TLS and Authorization
+^^^^^^^^^^^^^^^^^^^^^
+
 For production use cases, in particular if the Admin Api is not just bound to localhost, we recommend to enable
 :ref:`TLS <tls-configuration>` with mutual authentication.
 
+It is possible to use the remote console in the installations that utilize authorization. In that case, it is
+necessary to provide a valid access token. This can be achieved by modifying the configuration or by adding
+an option to the remote console's launch command as in the following snippet:
```

Comment:
```suggestion
The remote console can be used in installations that utilize authorization, so long as it has a valid access token. This can be achieved by modifying the configuration or by adding
an option to the remote console's launch command as in the following snippet:
```